.github/workflows/code-check-and-tests.yaml

```diff
@@ -77,6 +76,11 @@ jobs:
         with:
           args: --timeout=5m --config=/my/path/.golangci.yml
           version: v1.58
+      - name: yamllint
+        run: |
+          sudo apt-get install -y yamllint
+          yamllint --version
```

Comment:
```suggestion
```
`yamllint` is pre-installed on Github-hosted runners: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2004-Readme.md